src/main/java/com/google/alpollo/SentimentServlet.java

```diff
@@ -0,0 +1,27 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.alpollo.model.SongSentiment;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.gson.Gson;
+
+@WebServlet("/sentiment")
+public class SentimentServlet extends HttpServlet {
+  private final Gson gson = new Gson();
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String lyrics = request.getParameter("lyrics");
```

Comment:
maybe extract "lyrics" to a constant?

Reply:
I'm not sure I can make the variable static, it gives me the following error:
`Illegal modifier for parameter LYRICS; only final is permitted`
From my understanding, variables can be made static only outside of methods, but in my case i have to use the `request` parameter.

Reply:
Yea, you need to define it at class-level (field), not method-level.